wpinet/src/main/native/include/wpinet/MulticastServiceResolver.h

```diff
@@ -32,6 +32,17 @@ class MulticastServiceResolver {
     /// Service data payload.
     std::vector<std::pair<std::string, std::string>> txt;
   };
+  /**
+   * Set a copy callback to be called when a service is resolved.
+   * Takes presidence over the move callback. Return true to
+   * not send the data to the event queue.
+   */
+  bool SetCopyCallback(std::function<bool(const ServiceData&)> callback);
+  /**
```

Comment:
Should have a newline before each of these comment blocks.

Reply:
For some reason this is what the formatter did. Not sure why.

Reply:
The `#ifdef __cplusplus` around around the whole file is confusing wpiformat's #include parser.